### PR TITLE
Report css 2

### PR DESCRIPTION
**Pull request by adamhooper**

### CODE REVIEW

File: assets/css/WorkflowEditor/Report/_Block.scss [hunk 1]
```diff
@@ -1,17 +1,32 @@
 article.report .report-container section.block {
+  padding: 1rem 2rem;
   display: flex;
-  margin: 1rem 2rem;
+  flex-direction: column;
```
Comment (adamhooper) [inline assets/css/WorkflowEditor/Report/_Block.scss:4]: Please nix `display: flex` and `flex-direction: column`. Or add a comment saying why that doesn't work.

File: assets/css/WorkflowEditor/Report/_Block.scss [hunk 2]
```diff
@@ -1,17 +1,32 @@
...
 
+  }
   .block-actions {
     flex: 0 0 auto;
     display: flex;
-    flex-direction: column;
     justify-content: flex-end;
-    margin: 0 1rem 0 -1rem;
+    button {
+      font-size: 2rem;
```
Comment (adamhooper) [inline assets/css/WorkflowEditor/Report/_Block.scss:15]: This font-size affects the `<IconButton>`, and that makes things blurry. Please nix this `font-size` so the `<IconButton>` can be 16px.

File: assets/css/WorkflowEditor/Report/_Block.scss [hunk 3]
```diff
@@ -1,17 +1,32 @@
...
+      margin-left: .5rem;
+    }
+  }
+  &:hover {
+    background-color: $hover-background;
+    button.icon-button {
+      opacity: 1;
```
Comment (adamhooper) [inline assets/css/WorkflowEditor/Report/_Block.scss:22]: This shouldn't be in `&:hover` because the fact that you're hovering doesn't seem to change anything. Can you please delete the `opacity: 1` and move the `background-color` and `color` into `.block-actions button`?

File: assets/css/WorkflowEditor/Report/_Block.scss [hunk 4]
```diff
@@ -1,17 +1,32 @@
...
     justify-content: flex-end;
-    margin: 0 1rem 0 -1rem;
+    button {
+      font-size: 2rem;
+      margin-left: .5rem;
+    }
+  }
+  &:hover {
+    background-color: $hover-background;
+    button.icon-button {
```
Comment (adamhooper) [inline assets/css/WorkflowEditor/Report/_Block.scss:21]: Please change `button.icon-button {` to `.block-actions button {`: that's the selector we use elsewhere in this file, and using the same selector prevents CSS priority rules from doing something unexpected.

File: assets/css/WorkflowEditor/Report/_TextBlock.scss [hunk 5]
```diff
@@ -1,21 +1,26 @@
...
+      background-color: $Vlight-gray;
+
     }
     .buttons {
       display: flex;
       justify-content: flex-end;
 
       button {
-        margin: .5em;
+        margin: 1em .5em 0 0;
```
Comment (adamhooper) [inline assets/css/WorkflowEditor/Report/_TextBlock.scss:23]: Why is this here? Doesn't the margin in `_Block.scss` do what we want?

File: assets/icons/chart.svg [hunk 6]
```diff
@@ -1,87 +1,7 @@
...
-     id="path10"
-     sodipodi:nodetypes="ccccc" />
-  <path
-     fill="#686768"
-     d="m 10,12 h 4 v 12 h -4 z"
-     id="path12"
-     sodipodi:nodetypes="ccccc" />
+<svg width="32" height="32" viewBox="0 0 32 32" fill="none" xmlns="http://www.w3.org/2000/svg">
+<rect x="3" y="26" width="26" height="3" fill="#686768"/>
+<rect x="3" y="9" width="4.16" height="14" fill="#686768"/>
```
Comment (adamhooper) [inline assets/icons/chart.svg:3]: This icon isn't pixel-aligned. ("4.16" isn't an integer.) Please make it pixel-aligned?

File: assets/css/components/Fonts.scss [hunk 7]
```diff
@@ -92,6 +92,13 @@
   @include info-3;
 }
 
+@mixin report-content { // used in report
```
Comment (adamhooper) [inline assets/css/components/Fonts.scss:95]: Please nix this. It isn't used anywhere.

File: assets/icons/deprecated/table.svg [hunk 8]
```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (adamhooper) [inline assets/icons/deprecated/table.svg:1]: Please don't add or edit any icons in the `deprecated/` directory. That directory is designed to mimic our existing icon font. The only thing we'll ever do here is delete files once we stop using them (and thus no longer need them in the icon font).

File: assets/icons/table.svg [hunk 9]
```diff
@@ -1,67 +1,3 @@
...
-     id="path4"
-     sodipodi:nodetypes="cccccccccccccccccccccccccccccccccccccccccccccccccc" />
+<svg width="32" height="32" viewBox="0 0 32 32" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path fill-rule="evenodd" clip-rule="evenodd" d="M2.87509 2C2.35019 2 2 2.37314 2 2.93344V29.0666C2 29.6269 2.35019 30 2.87509 30H29.1249C29.6498 30 30 29.6269 30 29.0666V2.93344C30 2.37314 29.6498 2 29.1249 2H2.87509ZM3.70557 3.80293H7.48841V7.83799H3.70557V3.80293ZM8.43423 3.80293H17.8916V7.83799H8.43423V3.80293ZM18.8371 3.80293H28.2944V7.83799H18.8371V3.80293ZM3.70557 8.84687H7.48841V12.8819H3.70557V8.84687ZM8.43423 8.84687H17.8916V12.8819H8.43423V8.84687ZM18.8371 8.84687H28.2944V12.8819H18.8371V8.84687ZM3.70557 13.8904H7.48841V17.9259H3.70557V13.8904ZM8.43423 13.8904H17.8916V17.9259H8.43423V13.8904ZM18.8371 13.8904H28.2944V17.9259H18.8371V13.8904ZM3.70557 18.9343H7.48841V22.9694H3.70557V18.9343ZM8.43423 18.9343H17.8916V22.9694H8.43423V18.9343ZM18.8371 18.9343H28.2944V22.9694H18.8371V18.9343ZM3.70557 23.9783H7.48841V28.0133H3.70557V23.9783ZM8.43423 23.9783H17.8916V28.0133H8.43423V23.9783ZM18.8371 23.9783H28.2944V28.0133H18.8371V23.9783Z" fill="#686768"/>
```
Comment (adamhooper) [inline assets/icons/table.svg:2]: Please pixel-align.

File: assets/icons/text.svg [hunk 10]
```diff
@@ -1,70 +1,7 @@
...
-  </sodipodi:namedview>
-  <title
-     id="title2">text</title>
-  <path
-     id="path12"
-     style="stroke-width:0.687502"
-     d="M 5.4648438 2 C 4.6549619 2 3.9995379 2.656922 4 3.4667969 L 4 5 L 4 9 L 7 9 L 7 5 L 14 5 L 14 21 L 10 21 L 10 24 L 20 24 L 20 21 L 16 21 L 16 5 L 23 5 L 23 9 L 26 9 L 26 5 L 26 3.4667969 C 26 2.6569219 25.343084 2 24.533203 2 L 5.4667969 2 L 5.4648438 2 z " />
+<svg width="32" height="32" viewBox="0 0 32 32" fill="none" xmlns="http://www.w3.org/2000/svg">
+<rect x="3" y="4.73334" width="3.46667" height="6.93333" fill="#686768"/>
+<rect x="25.5333" y="4.73334" width="3.46667" height="6.93333" fill="#686768"/>
```
Comment (adamhooper) [inline assets/icons/text.svg:3]: Please pixel-align.

**Review by adamhooper — COMMENTED** [review]
I like the new look! Could you please fix according to the comments I've left?

File: assets/icons/chart.svg (hunk 6 above)
Comment (pierreconti) [reply assets/icons/chart.svg:3]: old one, I'll replace it

File: assets/icons/deprecated/table.svg (hunk 8 above)
Comment (pierreconti) [reply assets/icons/deprecated/table.svg:1]: Ok

File: assets/css/WorkflowEditor/Report/_Block.scss [hunk 11]
```diff
@@ -1,17 +1,35 @@
 article.report .report-container section.block {
-  display: flex;
-  margin: 1rem 2rem;
+  padding: 0 rem(2) 0 rem(1.7); // no padding at the top -- it's provided by `block-actions` // 1.7 left for highlight margin
+  border-left: rem(.3) solid transparent; // in order to highlight the block when hover
 
   .block-main {
     flex: 1 1 auto;
+    textarea {
```
Comment (adamhooper) [inline assets/css/WorkflowEditor/Report/_Block.scss:7]: TODO explain + place somewhere

File: templates/account/logout.html [hunk 12]
```diff
@@ -5,12 +5,12 @@
 {% block title %}{% trans_html "py.templates.account.logout.title" default="Log out" %}{% endblock %}
 
 {% block content %}
-    <form class="account_form" method="POST" action="{% url "account_logout" %}">
+    <form class="account_form logout" method="POST" action="{% url "account_logout" %}">
```
Comment (adamhooper) [inline templates/account/logout.html:8]: This is https://www.pivotaltracker.com/story/show/175882795 and we won't solve it in this pull request.

**Review by adamhooper — COMMENTED** [review]
Hey @pierreconti,

I've incorporated what I could into `master`. For everything else, I've left comments.

Please incorporate these comments into whatever list you need, then delete this pull request.
